jwst/pipeline/outlier_detection.cfg

```diff
@@ -14,4 +14,4 @@ scale = '0.5 0.4'
 backg = 0.0        
 save_intermediate_results = False
 resample_data = True
-good_bits = 6
+good_bits = 'SATURATED, JUMP_DET'
```

Comment:
This isn't consistent with what's in the step code "spec" block.

Reply:
DIdn't change this behavior to reduce the number of test failures; there would be enough from the regressions. As such, being "unit" tests, and being hard-wired, was not clear that the intent would not have been to set explicitly. Regardless, if the real need is to be "default", it can be made so.